examples/vmi-resource-claim.yaml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
The examples are generated from code. So, this fine to give an example, but it will be removed by the automatic tooling.

Reply:
I see . Is the code generation tool part of kubevirt ?

Reply:
yes